runtime/packages/core-modules/assembly/system.ts

```diff
@@ -58,5 +64,47 @@ export class System {
         const decodedHeader = Header.fromU8Array(result);
         return decodedHeader.result;
     }
+    /**
+     * Get the nonce value of the given AccountId
+     * The format of the key for storing nonce in the storage is:
+     * SCALE(AccountId) + SCALE("nonce")
+     * @param who nonce of this account
```

Comment:
`who nonce of this account` -> `who - account for which to get the nonce` or smth like that, as `who nonce of this account` seems confusing